LEAD_MAINTAINER_PROTOCOL.md

```diff
@@ -1,51 +1,66 @@
-## Tech Leads and Lead Maintainers
+## Lead Maintainer Protocol
 
-> This section is a formalization of the proposal and discussion around the [Module Lead Maintainers - Sharing the Responsibility over the IPFS module base](https://github.com/ipfs/pm/issues/600) proposal.
+> This section is a formalization of the proposal and discussion around the [Module Lead Maintainers - Sharing the Responsibility over the IPFS module base](https://github.com/ipfs/pm/issues/600) proposal, originally desined for the JS Ecosystem within the IPFS Project.
 
-**Currently, this Protocol is only being executed by the JS ecosystem of the IPFS Project.**
+## Motivation
 
-We have two types of leads in the JavaScript project ecosystems, the Tech Lead and the Module Lead Maintainer. A brief description of both roles is:
```

Comment:
It seems like you are removing the overall "Tech Lead" role and only having Lead Maintainers per-project or per-working-group. Is this also part of the intent of this change? Is the Tech-Lead role previously filled by you and @whyrusleeping going to be deprecated in lieu of Working Group Captains?

Reply:
Thank you for asking this question! I can understand why this PR would give that idea, however, it's intent is far from changing the definition of Tech Lead and who wears that hat. I believe that will have to be another discussion.

What this PR proposes is that we make the _Maintainer_ role something to be Org wide so that every project benefits from having a DRI and good Repo grooming practices.

Reply:
I still find all the different titles confusing - is this the appropriate breakdown:
- **Project creator** (@jbenet) - owns and communicates the vision, has the final say on escalations, the buck stops here
- **IPFS Project Captain** (@daviddias) - steers the org, master architect/strategist, can settle disagreements between individual WG Leads/Captains
- **Working Group Tech Lead** (Captain?) - steers the working group effort, manages vision and execution for many repos. Can resolve disputes / guide direction for individual module maintainers.
-  **Module Lead Maintainer** - main shepherd for a particular module including pruning, improvement, and incorporating contributions

aka - the "lead maintainer" title is focused specifically at the repo level, though tech leads / captains are effectively doing the same role for a broader project?

Reply:
> aka - the "lead maintainer" title is focused specifically at the repo level, though tech leads / captains are effectively doing the same role for a broader project?

That is correct. This question eventually takes us to what is the Governance of the protocol/project. However, I was hoping we can have that discussion separately as I believe it will require its own pace of review.

Meanwhile, I hope to provide more clarity with my last commit -- https://github.com/ipfs/team-mgmt/pull/822/commits/53379107340b7e6d78b1f5d7eca93436117c3cd0 -- in which I added to the title of the protocol "Repo", so that it is more clear that this is a "Repo Lead Maintainer" protocol and not a "Project Maintainer/Leader/Sheppard" protocol.

Sidenote: The reason why to used the word _Repo_ and not _Module_ is just to accommodate that we have repos that are not code and in some languages, a software unit is called a package and not a module. To accommodate everyone, we can use the word _Repo_ which is common to all.